integration_tests/build.gradle.kts

```
@@ -1,7 +1,7 @@
 plugins {
     java
     kotlin(Plugins.Kotlin.PLUGIN_JVM)
-    id(Plugins.DETEKT_PLUGIN) version Versions.DETEKT
+    id(Plugins.DETEKT_PLUGIN)
 }
 
```

```suggestion
tasks.withType<KotlinCompile> {
    dependsOn(tasks["detekt"])
}
```

That would run detekt task before compilation which could be more efficient when problems were found

---

skipping